VAMobile/src/components/FormWrapper/FormFields/VATextInput.test.tsx

```diff
@@ -92,21 +92,22 @@ context('VATextInput', () => {
   describe('when there is helper text', () => {
     it('should display it', async () => {
       initializeTestInstance('email', '', 'back.a11yHint')
-      expect(testInstance.findAllByType(TextView)[1].props.children).toEqual('Navigates to the previous page')
+      expect(screen.getByText('Navigates to the previous page')).toBeTruthy()
+      expect(testInstance.findByProps({ 'children': 'Navigates to the previous page' })).toBeTruthy()
```

Comment:
test instance, component should be removed. the initialize correctly test should also be removed

Reply:
Updated to use `screen` rather than `testInstance`.